ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -731,4 +745,10 @@
         "OTHERS": GATHER_COMMANDS_OTHERS + GATHER_COMMANDS_OTHERS_4_6,
         "OTHERS_EXTERNAL": GATHER_COMMANDS_OTHERS_EXTERNAL_4_6,
     },
+    4.7: {
```

Comment:
Are you also planning to make the new noobaa pod name changes in this PR?
Noobaa DB pod from 4.7 changed - from noobaa-db-0 to noobaa-db-pg-0